README.md

```diff
@@ -14,14 +14,14 @@ alternative command
 cnpm install gifsicle
 ```
 
-If you use **Debian/Ubuntu**, you need install ```gifsicle```:
+If you use **Debian/Ubuntu**, you need install ```dh-autoreconf```:
```

Comment:
Maybe just write `install`.
Otherwise it should be: `you need to install`
The same applies below for Mac.